src/odemis/dataio/tiff.py

```diff
@@ -2076,7 +2085,18 @@ def export(filename, data, thumbnail=None, compressed=True, multiple_files=False
     compressed (boolean): whether the file is compressed or not.
     multiple_files (boolean): whether the data is distributed across multiple
       files or not.
+    pyramid: whether to export data as pyramid
+    ome_compat: whether to export the file in OME-2016 format (in addition to standard metadata)
     '''
+
+    if ome_compat:
+        try:
+            new_fn = filename.replace(".ome.tiff", "-2016-06.ome.tiff")
```

Comment:
Let's not change the file name. It should respect what is passed as input. Actually, I think we should just by default output TIFF file compliant with OME-TIFF 201606. 
There could be a separate "exporter" which output ImageJ compatible TIFF. That would be some ".ij.tiff" format, with an explicit extension defined. I think it could also contain OME-TIFF 201606 metadata (as for ImageJ, all that is needed is a short INI-style header, and it can be followed by an XML data).
The only thing we need to be careful is that the old OME-TIFF files generated by Odemis can still be read.